### PR TITLE
Rewrite the README

### PR DESCRIPTION
**Pull request by pradyunsg**
Well, I felt the README could be better. So, I decided to rewrite it. Almost every sentence has been re-written and all the supporting images have been added to the repository.

---
- [x] Introduction
- [x] Features
- [x] Installation
- [x] Usage
  - [x] Supported Color Formats
    - [x] Image
  - [x] Color Picker usage
    - [x] Image
  - [x] Color Format Converter usage
  - [x] Less/SASS/SCSS/Stylus variables navigation
  - [x] Variables file
- [x] Settings
- [x] Contributing

### CODE REVIEW

**Comment by pradyunsg** [comment]
@Monnoroch Could you make 2 more images for Color Formatter and Variable Navigation? I'm not really able to figure out how to do that.

**Comment by Monnoroch** [comment]
@pradyunsg Nope, donno how to do that either.

**Comment by Monnoroch** [comment]
Is this still one still ongoing? I'm not so good in writing readmes, so this one probably would help alot.

**Comment by pradyunsg** [comment]
Well, other than those two images, I'm almost done... Just need to write another section...

**Comment by pradyunsg** [comment]
I'm done.

**Comment by Monnoroch** [comment]
I feel that gutter icons and highlighting all colors in a file are the key features of the plugin, so it would probably be good to tell about those somewhere in the beginning. 
Also, there was lots of pictures, and you only left gifs which a great, but don't show all the goodness (icons, etc).

**Comment by pradyunsg** [comment]
Sounds fine to me. But someone (not me) needs to make a bunch of GIFs, for these things:

 - [ ] Gutter Icons
 - [ ] Highlighting All Colours
 - [ ] Color Formatter
 - [ ] Variable Navigation

I'll add them up in the README then.

**Comment by pradyunsg** [comment]
Bump. @Monnoroch Any chance of you making those GIFs?

I'll rebase and rewrite.

**Comment by pradyunsg** [comment]
Bump 2.0

**Comment by davmillar** [comment]
I just did videos of the use cases that need GIFs and will convert them and make a pull request against the branch from @pradyunsg tomorrow.

**Comment by pradyunsg** [comment]
I'll fix the conflicts...

**Comment by davmillar** [comment]
@pradyunsg any ETA on the merge conflicts? If you're not able to fix the conflicts within a few days, I can help.

**Comment by pradyunsg** [comment]
@davmillar Please do. :)

**Comment by pradyunsg** [comment]
Ohkay, this is done.

**Comment by pradyunsg** [comment]
@Monnoroch @davmillar Ping?

**Comment by davmillar** [comment]
lgtm :+1:

File: README.md [hunk 1]
```diff
@@ -1,64 +1,123 @@
...
+ [package-control]: https://packagecontrol.io/installation
+ [imagemagick-url]: http://www.imagemagick.org/
 
-Just put cursor on a variable, right click on it and press "Go to variable definition" and the plugin will open it. There is also a shortcut <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>D</kbd>.
+ [image-color-picker]: images/color-picker.gif
+ [image-color-converter]: images/convert.gif
+ [image-underline-preview]: images/underline-preview.gif
+ [image-colored-text-preview]: images/colored-text-preview.gif
+ [image-gutter-dots-preview]: images/dots.gif
+ [image-go-to-definition]: images/go-to-definition.gif
```
Comment (Monnoroch) [inline README.md]: Did you create this GIF yourself? Is it possible to remove the bottom part with the wallpaper that is not useful? Same goes for convert.gif.

**Comment by Monnoroch** [comment]
Thank you very much for the amazing work! I've left a comment about gifs, but otherwise everything's looks good.

**Comment by pradyunsg** [comment]
@Monnoroch Could you merge this in?

@davmillar or I would make another PR to update the GIFs soon. :)

**Comment by Monnoroch** [comment]
@pradyunsg I could certainly merge it, but I would prefer to wait for this PR to be completed with better GIFs. There's no rush and it's always better to merge a finished and polished chunk of work that can ve reviewed as a whole.

**Comment by pradyunsg** [comment]
I didn't do anything with the GIFs so I don't know how long it'll be till someone comes around to make that change or I get around to installing stuff on my system to edit the GIFs.

I'd say this is already a big enough improvement over the current readme and the specific issue of the GIFs being slightly off-perfect, shouldn't be a blocker for this.

> I could certainly merge it

Please do. :)

**Comment by pradyunsg** [comment]
@davmiller Since you had contributed the GIFs, could you crop these?

**Comment by davmillar** [comment]
@pradyunsg @Monnoroch  so sorry, it looks like I started a comment but never submitted it so it remained as a draft. Just had a question about the cropping requested, please see above.

File: README.md (hunk 1 above)
Comment (pradyunsg) [reply README.md]: Ping.

**Comment by Monnoroch** [comment]
@davmillar @pradyunsg 

Hey guys! I'm really sorry, but because I've rewritten the plugin completely this PR is no longer mergeble.  However, I would still be very grateful if you would help me to make the new README awesome! Right now it lacks nice screenshots, including that of new features, GIFs etc.

**Comment by pradyunsg** [comment]
Hi @Monnoroch! I have my hands full right now so I don't really have much free time to do this. Sorry! :)

**Pull request closed without merging**